### ISSUE TITLE
Blank Query Search on a Sub Category

### ISSUE THREAD

**Issue by sammyhancock** [Prowlarr/Prowlarr#118]
**Is your feature request related to a problem? Please describe.**
Not related to a problem with the app, just that blank queries aren't currently supported when manually doing a query within Prowlarr.

**Describe the solution you'd like**
As in NZBHydra2, entering no query (a blank query) should output results for everything under the current category selected. NZBHydra2 also limits this blank query initially to so many shown instead of everything until the last date available from the indexer. You can also load up all results if you want to. It also has filters to limit how far back to query, the size, etc. I don't always specifically know what I'm looking for and I do a lot of manual blank queries to look at what's available within NZBHydra2 instead of opening up multiple indexer sites and looking at the categories there. This allows me to query multiple indexers that specialize in specific content while doing it from a single place.

**Describe alternatives you've considered**
Basically NZBHydra2 is my only alternative for the time being or searching using a year to pull up latest content.

**Additional context**
![image](https://user-images.githubusercontent.com/1185160/120940160-e6fdbb80-c6e9-11eb-895e-b7901e4898ed.png)
This screenshot shows the filters you can use within hydra

![image](https://user-images.githubusercontent.com/1185160/120940189-0d235b80-c6ea-11eb-8452-762d0f60b7c5.png)
This screenshot shows what happens when you do a blank query

**Comment by Qstick** [comment]
This should work now
